docs_src/security/Ch-APIGateway.md

```diff
@@ -282,35 +284,24 @@ resources behind the API gateway. Again, without the API Gateway in
 place, here is the sample request to hit the ping endpoint of the EdgeX
 Core Data microservice using curl:
 
-    curl http://<host-system-ip>:59880/api/v2/ping
+    curl http://<core-data-microservice-ip>:59880/api/v2/ping
 
 With the security service and JWT authentication is enabled, the command
 changes to:
 
-    curl -k --resolve kong:8443:127.0.0.1 -H 'Authorization: Bearer <JWT>' https://kong:8443/core-data/api/v2/ping
+    curl -k -H 'Authorization: Bearer <JWT>' https://myhostname:8443/core-data/api/v2/ping
 
-(Note the above `--resolve` line forces "kong" to resolve to 127.0.0.1.
-This is just for illustrative purposes to force SNI. In practice,
-the TLS certificate would be registered under the external host name.)
 
 In summary the difference between the two commands are listed below:
 
--   --resolve tells curl to resolve https://kong:8443 to
-    the loopback address.  This will cause curl to use the
-    hostname `kong` as the SNI, but connect to the specified IP
-    address to make the connection.
-    -k tells curl to ignore certificate errors. This is for
+-   `-k` tells curl to ignore certificate errors. This is for
     demonstration purposes. In production, a known certificate that
     the client trusts be installed on the proxy and this parameter omitted.
--   --H "host: edgex" is used to indicate that the request is for
-    EdgeX domain as the API gateway could be used to take requests for
-    different domains.
 -   Use the https versus http protocol identifier for SSL/TLS secure
     communication.
 -   The service port 8443 is the default TLS service port of API gateway
--   Use the URL path "coredata" to indicate which EdgeX microservice
+-   Use the URL path "core-data" to indicate which EdgeX microservice
     the request is routed to
--   Use header of -H "Authorization: Bearer \<access-token\>" to
-    specify the access token associated with the client that was
-    generated when the client was added.
+-   Use header of `-H "Authorization: Bearer \<JWT\>"` to
```

Comment:
```suggestion
-   Use header of `-H "Authorization: Bearer <JWT>"` to
```

Reply:
Fixed in https://github.com/bnevis-i/edgex-docs/commit/2fff06e6ba4522a0ffa6a91f3b1dc29a138b3c95